Corgibytes.Freshli.Cli/Program.cs

```diff
@@ -91,6 +97,19 @@ public static CommandLineBuilder CreateCommandLineBuilder()
                 await next(context);
             })
             .AddMiddleware(async (context, next) => { await LogExecution(context, next); })
+            .AddMiddleware(async (context, next) =>
+            {
+                var workerCount = context.ParseResult.GetOptionValueByName<int>("workers");
+                HangfireBackgroundService?.StopAsync(CancellationToken.None);
+                if (HangfireOptions != null && workerCount > 0)
+                {
+                    HangfireOptions.WorkerCount = workerCount;
+                }
+
+                HangfireBackgroundService?.StartAsync(new CancellationToken());
```

Comment:
Oh! This reminds me of an issue that we need to create. If I remember correctly, we're not able to stop the program with `Ctrl-C` once it's running. We should have a way to "listen" for that keypress and then signal the `CancelationToken` when it happens.

Reply:
Done #321 :D